code/game/gamemodes/game_mode.dm

```diff
@@ -436,9 +436,6 @@
 				msg += "<b>[L.name]</b> ([L.ckey]), the [L.job] (<font color='#ffcc00'><b>Connected, Inactive</b></font>)\n"
 				continue //AFK client
 			if(L.stat)
-				if(L.suiciding)	//Suicider
-					msg += "<b>[L.name]</b> ([L.ckey]), the [L.job] (<span class='boldannounce'>Suicide</span>)\n"

```

Comment:
As far as I can tell that status only occurs when someone commits suicide but is still in their body.  This isn't really a normal state to be in. In this pull the guy who burned his hands 20 times on a light and the guy who blew up an airtank are treated equal until one of them says "yeah peace I'm out!" and uses the verb.